### PR TITLE
Integrate xloader

### PR DESCRIPTION
**Pull request by RabiaSajjad**
# Overview

Please replace this line with full information about your pull request. Make sure that tests pass before publishing it

---

Please preserve this line to notify @amercader (lead of this repository)

### CODE REVIEW

File: ckanext/validation/jobs.py [hunk 1]
```diff
@@ -87,6 +88,14 @@ def run_validation_job(resource):
...
         validation.status = u'success' if report[u'valid'] else u'failure'
         validation.report = report
+
+        # load successfully validated resources to datastore using xloader
+        if validation.status == u'success':
+            if 'xloader' in config.get('ckan.plugins'):
+                t.get_action('xloader_submit')(
+                    {'ignore_auth': True,
+                     'user': t.get_action('get_site_user')({'ignore_auth': True})['name']},
+                    {'resource_id': resource['id']})
```
Comment (wardi) [inline ckanext/validation/jobs.py]: you might want to do this after the resource_patch below just to be safe

File: ckanext/validation/jobs.py [hunk 2]
```diff
@@ -108,9 +117,16 @@ def run_validation_job(resource):
...
+
+    use_proxy = 'ckan.download_proxy' in t.config
+    if use_proxy:
+        proxy = t.config.get('ckan.download_proxy')
+        log.debug(u'Download resource for validation via proxy: %s', proxy)
+        http_session.proxies.update({'http': proxy, 'https': proxy})
+    report = validate(source, format=_format, schema=schema, http_session=http_session, **options)
 
-    log.debug(u'Validating source: {}'.format(source))
+    log.debug(u'Validating source: %s', source)
```
Comment (wardi) [inline ckanext/validation/jobs.py:131]: was this from merging upstream? it doesn't look related

Comment (RabiaSajjad) [reply ckanext/validation/jobs.py:131]: yeah, from upstream https://github.com/open-data/ckanext-validation/commit/37ef0bac12c924248cb7ea523fa56b47cfe76a97
